### DISCUSSION TITLE
Announcing the iTwin.js Steering Committee

### DISCUSSION THREAD

**Discussion by granvilj** [iTwin/itwinjs-core#3104] [Announcements]
iTwin.js is an open-source library that provides a technical foundation that reduces development costs and time to market for digital twin solutions. Bentley has dedicated over five years and the work of hundreds of software engineers and subject matter experts to evolve the platform to where it is today.  This powerful enabling technology is helping accelerate the development of new apps, and we’re seeing a growing number of companies moving from idea to MVP in as little as 2 weeks.  Bentley has stepped up and made this base technology available as open-source, and with the release of version 3.0 we want your help to ensure that it meets the needs of its users, now and in the future. 

### What is the responsibility of the committee?

Unlike other open-source projects like Node.js where the committee is responsible for everything from governance to marketing, the iTwin.js Steering Committee has a singular remit to represent the voice of the developers. The committee works with iTwin.js maintainers to help guide development by ensuring the focus is on solving problems for the benefit of the iTwin.js community.  For a full explanation of the committee’s responsibilities go to the [iTwin.js Steering Committee Charter](https://github.com/iTwin/itwinjs-core/blob/master/Steering.md).

The only way to understand the needs of the community is to be part of the community and to that end all committee members must be actively developing applications or solutions using iTwin.js.

### Interested in becoming a member?

The first steering committee meeting will be held in March.  If you would like to join the committee, please send an email to Sheena.Gaynes@bentley.com with the subject line: "iTwin.js Steering Committee Application".  Let us know the company you work for and the applications your team is developing.

### iTwin.js Community

The steering committee is not the only way to have your voice be heard. If you have requests or questions, visit the [iTwin.js project on GitHub](https://github.com/iTwin) and contribute to our [discussion](https://github.com/iTwin/itwinjs-core/discussions) and [issue](https://github.com/iTwin/itwinjs-core/issues) forums.

**Comment by colemant** [comment]
look forward to to working with the committee
